frontend/lib/src/components/elements/PlotlyChart/PlotlyChart.tsx

```diff
@@ -305,10 +339,44 @@ function PlotlyFigure({
 
   const { data, layout, frames } = spec
 
+  const handleRelayout = (event: PlotRelayoutEvent): void => {
+    if (event.dragmode) {
+      widgetMgr.setElementState(element.id, DRAGMODE, event.dragmode)
+    } else if (event["xaxis.range[0]"]) {
+      widgetMgr.setElementState(element.id, RANGE, event)
+    } else if (event["xaxis.autorange"]) {
```

Comment:
Can you add a quick comment here inline what these events are?

Reply:
Done